app/components/layers/useCommunityBoardBudgetRequestsLayer.client.tsx

```diff
@@ -158,11 +170,19 @@ export function useCommunityBoardBudgetRequestsLayer(opts?: {
       getIcon: [clickInfo.id, cbbrId],
       getIconSize: [clickInfo.id, cbbrId],
       getFillColor: [clickInfo.id, cbbrId],
+      highlightedFeatureId: [hoveredCbbr],
     },
-    onClick: (d) => {
-      setClickInfo({ id: d.object?.properties?.id, clicked: d.picked });
-
-      const indvidualCbbrId = d.object?.properties?.id;
+    onHover: (data) => {
+      const id = data.object?.properties?.id;
+      if (data.index === -1) {
+        setHoveredOverCbbr(null);
+      } else if (id) {
+        setHoveredOverCbbr(id);
+      }
+    },
+    onClick: (data) => {
+      setClickInfo({ id: data.object?.properties?.id, clicked: data.picked });
```

Comment:
Does the `picked` value toggle between `true` and `false`? From what I can tell,  we always check whether the project id matches the clicked id, and we check whether its "picked/clicked" status was true. We can simplify these other checks by resetting the 'clicked id` to "nothing" when "picked" is false.

this value reset becomes:

```ts
const id = data.object?.properties?.id;
const nextClickedId = data.picked && id !== undefined ? id : null;
setClickInfo(nextClickedId);
``` 

Refactoring the state:
(NULL is a more idiomatic expression of "no value" than an empty string)

```ts
[clickedId, setClickedId] = useState<string | null>(null)
```

The checks become:

```ts
      if (clickedId === id) {
```